git_machete/generated_long_docs.py

```diff
@@ -0,0 +1,960 @@
+
+        The format of the definition file should be as follows:
+        <dim>
+          develop
+              adjust-reads-prec PR #234
+                  block-cancel-order PR #235
+                      change-table
+                          drop-location-type
+              edit-margin-not-allowed
+                  full-load-gatling
+              grep-errors-script
+          master
+              hotfix/receipt-trigger PR #236
+        </dim>
+
+        In the above example `develop` and `master` are roots of the tree of branch dependencies.
+        Branches `adjust-reads-prec`, `edit-margin-not-allowed` and `grep-errors-script` are direct downstream branches for `develop`.
+        `block-cancel-order` is a downstream branch of `adjust-reads-prec`, `change-table` is a downstream branch of `block-cancel-order` and so on.
+
+        Every branch name can be followed (after a single space as a delimiter) by a custom annotation — a PR number in the above example.
+        The annotations don't influence the way `git machete` operates other than that they are displayed in the output of the `status` command.
+        Also see help for `anno` command.
+
+        Tabs or any number of spaces can be used as indentation.
+        It's only important to be consistent wrt. the sequence of characters used for indentation between all lines.
+   """,
+    "github": """
+        <b>Usage:</b><b>
+           git machete github <subcommand></b>
+
+        where `<subcommand>` is one of: `anno-prs`, `checkout-prs`, `create-pr`, `retarget-pr`.
+
+        Creates, checks out and manages GitHub PRs while keeping them reflected in branch definition file.
+
+        To allow GitHub API access for private repositories (and also to perform side-effecting actions like opening a PR, even in case of public repositories),
+        a GitHub API token with `repo` scope is required, see https://github.com/settings/tokens. This will be resolved from the first of:
+           * `GITHUB_TOKEN` env var,
+           * content of the `.github-token` file in the home directory (`~`),
+           * current auth token from the `gh` GitHub CLI,
+           * current auth token from the `hub` GitHub CLI.
+
+        GitHub API server URL will be inferred from `git remote`.
+        You can override this by setting the following git config keys:
+           Remote name
+              E.g. `machete.github.remote` = `origin`
+
+           Organization name
+              E.g. `machete.github.organization` = `VirtusLab`
+
+           Repository name
+              E.g. `machete.github.repository` = `git-machete`
+
+        To do this, run `git config --local --edit` and add the following section:
+        <dim>
+          [machete "github"]
+              organization = <organization_name>
+              repository = <repo_name>
+              remote = <remote_name>
+        </dim>
```

Comment:
I'd add some header here like `Subcommands`

Reply:
Coz in Python docs there's a sudden transition from the sample git config to `anno-prs`